CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## 0.60.0      2024-10-15
+
```

Comment:
Per the discussion in our meeting, we could add a notice like
```suggestion
WARNING: For Enterprise Server releases older than 2024.10 using [a separate collector installation](https://pganalyze.com/docs/enterprise/setup/separate-collector-install), this release is partially incompatible. Log Insights and Automated EXPLAIN will not receive any data if using this collector. Enterprise Server installations using an integrated collector are not affected, nor are Scale and Production plans.
```

Reply:
👍, but Automated EXPLAIN will still work so I removed that part